Food_Tracker_App/app/src/main/java/com/example/foodtracker/ui/recipes/AddRecipeActivity.java

```diff
@@ -58,13 +66,24 @@ protected void onCreate(Bundle savedInstanceState) {
         categoryField = findViewById(R.id.recipeCategory);
         categoryAdapter = new ArrayAdapter<>(this, android.R.layout.simple_dropdown_item_1line, categories);
         categoryField.setAdapter(categoryAdapter);
-        getCategories();
```

Comment:
You cannot remove this here since this will populate the categories in the dialog.